From 25d2a3db60a018f9af54154c7fe920f9e4f1cc97 Mon Sep 17 00:00:00 2001
From: Julie Daragnes <julie.daragnes@intel.com>
Date: Mon, 4 May 2015 17:37:48 +0200
Subject: [PATCH] REVERTME: xHCI: expose TRB error log

The patch is for detecting a modem issue: SMS06405729.
Crashtool relies on this log to report the modem issue.

Change-Id: Ib11cae61c1d0613e7297eef6b5bac28d14ee4717
Tracked-On: https://jira01.devtools.intel.com/browse/AKP-1535
Signed-off-by: Zhuang Jin Can <jin.can.zhuang@intel.com>
---
 drivers/usb/host/xhci-ring.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/host/xhci-ring.c b/drivers/usb/host/xhci-ring.c
index f46c1fc..48c4237 100644
--- a/drivers/usb/host/xhci-ring.c
+++ b/drivers/usb/host/xhci-ring.c
@@ -2116,7 +2116,7 @@ static int process_ctrl_td(struct xhci_hcd *xhci, struct xhci_td *td,
 		if (!xhci_requires_manual_halt_cleanup(xhci,
 					ep_ctx, trb_comp_code))
 			break;
-		xhci_dbg(xhci, "TRB error code %u, "
+		xhci_err(xhci, "TRB error code %u, "
 				"halted endpoint index = %u\n",
 				trb_comp_code, ep_index);
 		/* else fall through */
-- 
1.9.1

